src/utils/helpers.h

```diff
@@ -305,6 +305,12 @@ inline void check_tws(const std::vector<TimeWindow>& tws) {
   }
 }
 
+inline void check_priority(const Priority priority) {
+  if (priority > MAX_PRIORITY) {
+    throw Exception(ERROR::INPUT, "Invalid priority value.");
```

Comment:
Exceptions are changed simultaneously in #641, I'll adjust this myself in the merge commit.